jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Connection.java

```diff
@@ -323,16 +325,19 @@ public void onConnectionFailure(int error, String reason)
 
     protected class HTTP2Producer implements ExecutionStrategy.Producer
     {
+        private static final RetainableByteBuffer.Mutable STOPPED = new RetainableByteBuffer.NonRetainableByteBuffer(BufferUtil.EMPTY_BUFFER);
         private final Callback fillableCallback = new FillableCallback();
+        private final AtomicReference<RetainableByteBuffer.Mutable> savedBuffer = new AtomicReference<>();
```

Comment:
Don't like the word "saved" here, feels like saved to disk.
I'd use "retain", but it's overloaded.
I would use "heldBuffer".